pkg/fission-cli/cmd/spec/validate.go

```diff
@@ -63,6 +64,11 @@ func (opts *ValidateSubCommand) run(input cli.Input) error {
 	if err != nil {
 		return errors.Wrap(err, "error validating specs")
 	}
+	// check if any of the spec resource is already present in the cluster
+	error := ResourceExists(opts, fr)
+	if error != nil {
+		return errors.Wrap(error, " Spec validation failed")
```

Comment:
Please remove the whitespace from message. `" Spec validation failed"` -> `"Spec validation failed"`